db/seeds.rb

```diff
@@ -133,6 +133,8 @@ def url
   Attachment.create!(resource: kgp, file: Rack::Test::UploadedFile.new("spec/fixtures/mobile_only.png", "image/png"), is_zipped: true)
   Attachment.create!(resource: kgp, file: Rack::Test::UploadedFile.new("spec/fixtures/web_mobile.png", "image/png"))
   Attachment.create!(resource: kgp, file: Rack::Test::UploadedFile.new("spec/fixtures/both.png", "image/png"), is_zipped: true)
+  Attachment.create!(resource: kgp, file: Rack::Test::UploadedFile.new("spec/fixtures/web_attach.png", "image/png"))
+  Attachment.create!(resource: kgp, file: Rack::Test::UploadedFile.new("spec/fixtures/web_bundled.png", "image/png"))
```

Comment:
This doesn't seem to work for me locally, because line 130 fails

Reply:
would using `find_or_create_by` also work? testing it out locally right now

Reply:
Hmm, no... then I get an error on line 131 because that particular file already exists

Reply:
ok, I get why it's failing, let me try out a couple things. I think just changing it to check for the last file might lead to some other conflicts as it creates all the files again. (not sure if rails would protected against that)

Reply:
This is not a problem in the Action where the test db is created from scratch

Reply:
I was using `bundle exec rails db:reset` locally when testing, which was why I didn't see the issue of migrating the old seed db to the new seed db